Makefile

```diff
@@ -16,3 +16,6 @@ prepare: clean
 
 clean:
 	rm -rf demo
+
+authors:
+	git log --all --format='%aN <%aE>' | sort -u > AUTHORS
```

Comment:
It is interesting to update the contributors list automatically. But, as I already pointed in the previous comment, there is some flaws.
I would like to create a PR template instead. This template should asks to the contributor to update the `CONTRIBUTORS.md` file.

https://embeddedartistry.com/blog/2017/8/4/a-github-pull-request-template-for-your-projects

Reply:
The problem with manual updating the file (separate or as part of the README) is the it requires maintenance and people tend to not do it. We already have this issue as there are people who have contributed but their name is not on the list. That's why I tried to make this as automated as possible.